src/commands/Management/triggers.ts

```diff
@@ -119,17 +119,18 @@ export default class extends SkyraCommand {
 		}
 		if (!output.length) throw t(LanguageKeys.Commands.Management.TriggersListEmpty);
 
-		const display = new UserRichDisplay(
-			new MessageEmbed()
+		const display = new UserPaginatedMessage({
+			template: new MessageEmbed()
 				.setAuthor(message.author.username, message.author.displayAvatarURL({ size: 128, format: 'png', dynamic: true }))
 				.setColor(await DbSet.fetchColor(message))
-		);
+		});
 
 		for (const page of chunk(output, 10)) {
-			display.addPage((embed: MessageEmbed) => embed.setDescription(page));
+			display.addPageEmbed((embed) => embed.setDescription(page));
 		}
 
-		return display.start(message, undefined, { time: 120000 });
+		display.setIdle(120000);
+		return display.start(message);
```

Comment:
Use Time enum, or this
```suggestion
		return display.setIdle(120_000).start(message);
```